### DISCUSSION TITLE
Should I use just a single account for all of my organizations?

### DISCUSSION THREAD

**Discussion by eyalch** [getsentry/support#55] [Support] [answered]
I'm wondering if it makes sense to have 2 separate users - one for professional work projects (in my company) and another for personal projects. I see that one user can have multiple organizations, so I believe this should be possible.

As a reference, GitHub recommends having just a single account for both use cases: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/merging-multiple-personal-accounts

> **Tip:** We recommend using only one personal account to manage both personal and professional repositories.

What's Sentry's opinion on this matter?

**Comment by souredoutlook** [comment, marked as answer]
I had a look through our docs and while there is no official Sentry recommendation, using one account will allow you to easily manage your Sentry experience across multiple organizations without the need to manage two (or more) sets of [account preferences](https://sentry.io/settings/account/details/). 

My personal recommendation is to use one user account for multiple Sentry Organization memberships. Switching between organizations is quite simple and identity management (when different organizations have different auth requirements) saw recent quality-of-life enhancements.
